File: PROGLANG.py
# cook your dish here
for i in range(int(input())):
    a,b,a1,b1,a2,b2=list(map(int,input().split()))
    if a in (a1,b1) and b in (a1,b1):
        print(1)
    elif a in (a2,b2) and b in (a2,b2):
        print(2)
    else:
        print(0)

File: PROGLANG.cpp
#include <iostream>
using namespace std;

int main() {
    int t;
    cin>>t;
    while(t--){
        int ar[6];
        for(int i=0;i<6;i++){
            cin>>ar[i];
        }
        if((ar[0]==ar[2] || ar[0]==ar[3]) && (ar[1]==ar[2] || ar[1]==ar[3])){
            cout<<"1"<<endl;
        }
        else if((ar[0]==ar[4] || ar[0]==ar[5]) && (ar[1]==ar[4] || ar[1]==ar[5])){
            cout<<"2"<<endl;
        }
        else{
            cout<<"0"<<endl;
        }
    }
	// your code goes here
	return 0;
}
